packages/api/src/controllers/access-control.ts

```diff
@@ -30,12 +31,11 @@ import { DBStream } from "../store/stream-table";
 import { WithID } from "../store/types";
 
 const WEBHOOK_TIMEOUT = 30 * 1000;
-const MAX_ALLOWED_VIEWERS_FOR_HACKER_TIER_PER_NODE = 5;
+const DEFAULT_MAX_CONCURRENT_VIEWERS = 50_000;
```

Comment:
Shouldn't this and some of the plan limits on `config.js` be `10`?

Reply:
The new `Sandbox` limit (free tier) is 30 - all other plans are 50k. I added a default here in case for deprecated or not tracked plans (i.e. TheLRadio)